model/src/main/proto/spotlight.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Superclass for all the SpotlightViewStates
```

Comment:
Not technically a class, and this documentation could be slightly confusing. Consider, instead, explaining what a spotlight is (generally) and what this proto represents with respect to those.

Reply:
done, PTAL and lmk if changes are needed.